tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -230,6 +232,36 @@ func (ls *leaseStresser) createLeases() {
 	wg.Wait()
 }
 
+func (ls *leaseStresser) createShortTermLeases() {
+	neededLeases := ls.numLeases - len(ls.shortLivedLeases.getLeasesMap())
```

Comment:
we never revoke any short leases. they are being created for each test case. right now i just create 10 of them just to stress the etcd a bit. in the future, we probably can increase the for more stressing.